chat/message-rating/src/MessageRating/MessageRating.styles.ts

```diff
@@ -1,35 +1,60 @@
-import { css } from '@leafygreen-ui/emotion';
+import { css, cx } from '@leafygreen-ui/emotion';
 import { palette } from '@leafygreen-ui/palette';
 import { spacing } from '@leafygreen-ui/tokens';
 
-export const baseStyles = css`
+const baseContainerStyles = css`
   display: flex;
   align-items: center;
   gap: ${spacing[200]}px;
 `;
 
+export const getContainerStyles = (className?: string) =>
+  cx(baseContainerStyles, className);
+
 export const buttonContainerStyles = css`
   display: flex;
   align-items: center;
   gap: ${spacing[100]}px;
 `;
 
-export const getIconFill = (darkMode: boolean, isSelected: boolean) => {
-  if (darkMode) {
-    if (isSelected) {
-      return palette.black;
-    } else {
-      return palette.gray.light2;
-    }
-  } else {
-    if (isSelected) {
-      return palette.gray.light3;
-    } else {
-      return palette.gray.dark1;
-    }
+const getBaseIconFill = (darkMode: boolean) =>
+  palette.gray[darkMode ? 'light2' : 'dark1'];
+
+const getSelectedIconFill = (darkMode: boolean) =>
+  darkMode ? palette.black : palette.gray.light3;
+
+export const getIconFill = ({
+  darkMode,
+  isSelected,
+}: {
+  darkMode: boolean;
+  isSelected: boolean;
+}) => (isSelected ? getSelectedIconFill(darkMode) : getBaseIconFill(darkMode));
+
+const baseActiveStyles = css`
+  &:before {
```

Comment:
```suggestion
  &::before {
```

nit: `before` is a pseudo-element, so we should use double brackets `::before`. [Double brackets distinguish pseudo-elements from pseudo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements).